src/ray/core_worker/reference_count.cc

```diff
@@ -939,6 +939,38 @@ void ReferenceCounter::HandleObjectSpilled(const ObjectID &object_id) {
   ReleasePlasmaObject(it);
 }
 
+absl::optional<LocalityData> ReferenceCounter::GetLocalityData(
+    const ObjectID &object_id) {
+  absl::MutexLock lock(&mutex_);
+  // Uses the reference table to return locality data for an object.
+  auto it = object_id_refs_.find(object_id);
+  if (it != object_id_refs_.end()) {
+    // Reference for object_id exists.
```

Comment:
Really minor, but can you write comment => if instead? It could be probably just my personal preference though, so it is up to you. 

ex)
```cpp
  // Reference for object_id exists.
  if (it != object_id_refs_.end()) {
```